users/service.go

```diff
@@ -198,15 +203,32 @@ func (svc usersService) Login(ctx context.Context, user User) (string, error) {
 	return svc.issue(ctx, dbUser.Email, authn.UserKey)
 }
 
-func (svc usersService) User(ctx context.Context, token string) (User, error) {
+func (svc usersService) User(ctx context.Context, token, id string) (User, error) {
 	email, err := svc.identify(ctx, token)
 	if err != nil {
 		return User{}, err
 	}
+
+	// Retrieve User infos by ID if User is Admin and UUID is valid
```

Comment:
`User info`, not `User infos`. "Information" is uncountable noun, it does not nave plural.